notification/src/interfaces/sandboxExecutor.ts

```diff
@@ -1,3 +1,3 @@
 export default interface SandboxExecutor {
-  evaluate(expression: string, data: object): boolean;
+  evaluate(expression: string, data?: object): boolean;
```

Comment:
Why is data optional? I think we need data to evaluate the condition

Reply:
This adds some functionality to the platform.

I thought of the use case that the user might have only notifications sent to another platform if the pipeline/transormation fails (this might make more sense with tightly scheduled jobs)

In this case he might set the condition to `data === null`